src/main/kotlin/org/rust/ide/actions/macroExpansion/RsShowSingleStepMacroExpansionAction.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.actions.macroExpansion
+
+import org.rust.lang.core.macros.RsExpandedElement
+import org.rust.lang.core.psi.RsMacroCall
+import org.rust.lang.core.psi.ext.expansion
+
+/**
+ * Action for showing first-level expansion of ordinary macros (not procedural or custom derives).
+ */
+class RsShowSingleStepMacroExpansionAction : RsShowMacroExpansionActionBase(expandRecursively = false)
```

Comment:
I think it's ok to move these oneline classes to `RsShowMacroExpansionActionBase.kt` file